### ISSUE TITLE
Start using production ColdFront for OCP Prod

### ISSUE THREAD

**Issue by knikolla** [nerc-project/coldfront-nerc#70]
The new plan is to keep using ColdFront staging for the early adopters of the OCP Prod cluster. This card tracks the task to move from Staging ColdFront to Production Coldfront once we are ready to rollout to all users.

-- Original Description --
After #69 is up, all that's left is running the `create_openshift_resource` through the pod terminal.

**Comment by knikolla** [comment]
The acct-mgt interface is running on the route https://acct-mgt-acct-mgt.apps.nerc-ocp-prod.rc.fas.harvard.edu, ColdFront needs to have access to it.

**Comment by joachimweyl** [comment]
Please add me to a new epic.
